.github/workflows/build.yml

```
@@ -18,73 +18,75 @@ jobs:
-    - name: Build the geckodriver binaries in the artifacts directory
-      run: |
-        docker run --rm -v $PWD/artifacts:/media/host -w /opt/geckodriver local/geckodriver-cross-builder sh -c "sh build-arm.sh release $HOST_TRIPLE"
-      
-    - name: Package up the binaries in a tar.gz and calculate md5 hashes
-      run: |
-        sudo chown -R $USER:$USER artifacts
-        mv artifacts/geckodriver geckodriver
-        tar cvfz geckodriver-v$GECKODRIVER_VERSION-linux-$ARCH.tar.gz geckodriver
-        md5sum geckodriver-v$GECKODRIVER_VERSION-linux-$ARCH.tar.gz > geckodriver-v$GECKODRIVER_VERSION-linux-$ARCH.tar.gz.md5
+      - name: Package up the binaries in a tar.gz and calculate md5 hashes
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:15: Double quote to prevent globbing and word splitting

(shellcheck)